fdbclient/include/fdbclient/ClientKnobs.h

```diff
@@ -110,6 +110,7 @@ class ClientKnobs : public KnobsImpl<ClientKnobs> {
 	bool QUARANTINE_TSS_ON_MISMATCH;
 	double CHANGE_FEED_EMPTY_BATCH_TIME;
 	bool SHARD_ENCODE_LOCATION_METADATA;
+	bool DD_PHYSICAL_SHARD_CORE;
```

Comment:
nit: Consider changing to to `ENABLE_DD_PHYSICAL_SHARD`

Reply:
Changed. Thanks!